### PR TITLE
Update howdoi

### PR DESCRIPTION
**Pull request by dcherian**
Added a few more things.

### CODE REVIEW

**Review by max-sixty — APPROVED** [review]
Great, thanks @dcherian

File: doc/howdoi.rst [hunk 1]
```diff
@@ -59,3 +65,9 @@ How do I ...
      - ``obj.dt.ceil``, ``obj.dt.floor``, ``obj.dt.round``. See :ref:`dt_accessor` for more.
    * - make a mask that is ``True`` where an object contains any of the values in a array
      - :py:meth:`Dataset.isin`, :py:meth:`DataArray.isin`
+   * - Index using a boolean mask
+     - :py:meth:`Dataset.query`, :py:meth:`DataArray.query`, :py:meth:`Dataset.where`, :py:meth:`DataArray.where`
+   * - preserve ``attrs`` during xarray operations
```
Comment (mathause) [inline doc/howdoi.rst:70]:
```suggestion
   * - preserve ``attrs`` during (most) xarray operations
```

`(many)`?

File: doc/howdoi.rst [hunk 2]
```diff
@@ -59,3 +65,9 @@ How do I ...
...
+     - ``xr.set_options(keep_attrs=True)``
+   * - Make sure my datasets have values at the same coordinate locations
```
Comment (mathause) [inline doc/howdoi.rst:72]: Move next to the `make xarray objects look like other xarray objects` cell?

**Review by mathause — APPROVED** [review]

**Comment by dcherian** [comment]
The pre-commit hook is failing though the checks do pass
```
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...............................................................Passed
isort....................................................................Passed
black....................................................................Passed
blackdoc.................................................................Passed
flake8...................................................................Passed
mypy.....................................................................Passed

Error: reserveCache failed: Cache already exists. Scope: refs/pull/5114/merge, Key: pre-commit-2-74cc0ad84a16ba798fae779720b2bb7f287bfb9d3cb53e6e6d991fac1405b7d1-b8674f9d135cd10b0ed7dadee9b8a1499a33b9b32ec4cf985e6a0be67e2153dc, Version: 28cdb9f5496f334116f23e86f0063f5d3a9348c2e22425a33171e071aadada7e
```

**Comment by keewis** [comment]
for reference, this:
```
Error: reserveCache failed: Cache already exists. Scope: refs/pull/5114/merge, Key: pre-commit-2-74cc0ad84a16ba798fae779720b2bb7f287bfb9d3cb53e6e6d991fac1405b7d1-b8674f9d135cd10b0ed7dadee9b8a1499a33b9b32ec4cf985e6a0be67e2153dc, Version: 28cdb9f5496f334116f23e86f0063f5d3a9348c2e22425a33171e071aadada7e
```
can be fixed by rerunning the `pre-commit` action

**Pull request merged**